src/api/public/apidocs-new/paths/person.yaml

```diff
@@ -0,0 +1,27 @@
+get:
+  summary: List all persons.
```

Comment:
The plural of _person_ is **people**, not _persons_.

Reply:
So true :facepalm: Fixed